### PR TITLE
Document bundler options

### PR DESCRIPTION
**Pull request by mischnic**

### CODE REVIEW

File: src/features/code-splitting.md [hunk 1]
```diff
@@ -73,6 +71,45 @@ If a asset is imported both synchrounously and asynchrounously, it doesn't make
...
+- **minBundles**: for an asset to be split, it has to be used by more than `minBundles` bundles
+- **minBundleSize**: for a shared bundled to be created, it has to be at least `minBundleSize` bytes big (before minification/treeshaking)
+- **maxParallelRequests**: To prevent overloading the network with too many concurrent requests, this ensure that a given bundle can have only `maxParallelRequests - 1` sibling bundles (which have be loaded together with the actual bundle).
+
+- **http**: This is a shorthand for setting the above values to defaults which are optimized for HTTP/1 or HTTP/2:
+
+| HTTP version `version` | `minBundles` | `minBundleSize` | `maxParallelRequests` |
+| ---------------------- | ------------ | --------------- | --------------------- |
+| 1                      | 1            | 30000           | 6                     |
+| 2 (default)            | 1            | 20000           | 25                    |
```
Comment (surma) [inline src/features/code-splitting.md:111]: Unsolicited Q / backseat driving: Would it be possible to link to a source where it is explained why these numbers are beneficial?

Comment (mischnic) [reply src/features/code-splitting.md:111]: @devongovett ?

Comment (mischnic) [reply src/features/code-splitting.md:111]: @surma You might know the article I've linked to now 😄

Comment (surma) [reply src/features/code-splitting.md:111]: ahaha nice!

**Review by DeMoorJasper — APPROVED** [review]

**Pull request merged**